flowmap/particles.py

```diff
@@ -0,0 +1,121 @@
+import numpy as np
+import pandas as pd
+from tvtk.api import tvtk
+from tvtk.common import configure_input, configure_source_data, configure_input_data
+import vtk
+import geojson
+import pyproj
+
+def make_tracer_pipeline(polydata, seed):
+    # create elements of the pipeline
+    tracer = tvtk.StreamTracer()
+
+
+    # # # You can compute up to 1km per timestep (8km/hr for )
+
+    n = 200  # max number of steps
```

Comment:
Don't want these as arguments of the function?

Reply:
renamed for consistency and added to arguments.